src/reference-manual/expressions.Rmd

```diff
@@ -267,6 +267,8 @@ or variable names,  so it is legal to write
 `[ 2 * 3, 1 + 4]`  or `[ x, y ]`, providing that `x`
 and `y` are primitive variables.
 
+### Complex vector expressions {-}
```

Comment:
This blank section should be given a body or removed

Reply:
added a new section for complex vector and matrix expressions putting everything together.